dvc/config_schema.py

```diff
@@ -239,6 +239,9 @@ def __call__(self, data):
                     "ssl_verify": Any(Bool, str),
                     "token": str,
                     "use_https": Bool,
+                    "user": str,
+                    "password": str,
+                    "data_proxy_target": str,
```

Comment:
You'll need to bump fsspec version in dvc-webhdfs and then bump dvc-webhdfs version in this PR before we can merge it.

Reply:
Asked the maintainer when fsspec will have a new release that I can point to. As soon as I have that, will modify the dvc-webhdfs PR to include that version and then we can bump the version here.

Reply:
I think dvc-webhdfs already had a new release, but I don't think it was updated at that point to include the bumped version of fsspec.

Can you maybe help me to understand how I can add a requirement for fsspec to be >= 2023.12.0   for the new webhdfs?

I don't see an explicit fsspec dependency in dvc-webhdfs. Should I just create a pr that modifies the pyproject.toml of dvc-webhdfs to have a hard dependency there on >= 2023.12.0 ?